### DISCUSSION TITLE
Sobre feedback e vagas

### DISCUSSION THREAD

**Discussion by heliomsolivas** [frontendbr/forum#1232] [Discussão]
Então pessoal, acredito que não seja só eu que tenha passado por isso, mas os "recrutadores" que postam vagas no repositório entram em contato e acabam mandando um teste, você realiza e (pelo menos comigo), não recebe feedback nenhum.

Sei que não tem nada a ver com o pessoal do fórum, mas eu estava pensando se existe alguma espécie de "reclame aqui" para os "recrutadores" de empresas, porque é exigido tanta coisa de um futuro funcionário mas parece que existe um descaso com quem está buscando uma vaga.

**Comment by welyntonoliveira** [comment]
Então participei de alguns processos e não tive nenhum feedback, acho válido esse retorno, como você diz se pede tanto dos candidatos, a gente gasta tempo para não receber um nada isso é bastante injusto.
Essa ideia de um "reclame aqui" é muito válido para todos que já passaram por isso.

**Comment by didiraja** [comment]
Recomenda a issue #917 que movimentou bastante daqui
